src/bunyan-api.js

```diff
@@ -18,25 +18,35 @@ const bunyan = require('bunyan');
 
 module.exports = {
   'createLogger': function (name) {
+    
+    const loggerConfig = {
+      name: name || 'MustacheTemplate',
+      streams: [{
+        level: (process.env.LOG_LEVEL || 'info'),
+        stream: process.stdout // log LOG_LEVEL and above to stdout
+      }],
+      serializers: bunyan.stdSerializers,
+    };
+
     try {
-      return bunyan.createLogger({
-        name: name || 'MustacheTemplate',
-        streams: [{
-          level: (process.env.LOG_LEVEL || 'info'),
-          stream: process.stdout // log LOG_LEVEL and above to stdout
-        }],
-        serializers: bunyan.stdSerializers
-      });
+      if ( process.env.LOG_APPENDAGE ) {
+        try {
+          Object.assign(loggerConfig, JSON.parse( process.env.LOG_APPENDAGE ) );
```

Comment:
What format would `process.env.LOG_APPENDAGE`? 
If we really want it to be JSON, we need to parse separately and handle parse errors.

Reply:
Yes, it is JSON.  It is in its own try/catch block.